src/services/findAllReferences.ts

```diff
@@ -642,6 +642,12 @@ namespace ts.FindAllReferences {
             return mergeReferences(program, moduleReferences, references, moduleReferencesOfExportTarget);
         }
 
+        export function getReferencedSymbolsForFileName(fileName: string, program: Program, sourceFiles: readonly SourceFile[], sourceFilesSet: ReadonlySet<string> = new Set(sourceFiles.map(f => f.fileName))): readonly SymbolAndEntries[] | undefined {
+            const moduleSymbol = program.getSourceFile(fileName)?.symbol;

```

Comment:
dont you need to use fileRefMap if this is not module.. esp if file is not module ?

Reply:
To start with, I just wanted to get this working as well as find-all-refs on module specifiers, which only does a string search. It seems like we should use fileRefMap for that case too.

Reply:
This is done